Tools/2MGFX/SourceShaderLanguage.cs

```diff
@@ -0,0 +1,16 @@
+﻿// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+namespace TwoMGFX
+{
+    /// <summary>
+    /// Shading languages that 2MGFX supports.
+    /// </summary>
+    public enum SourceShadingLanguage
+    {
+        Unknown,
+        Hlsl,
+        Glsl
+    }
```

Comment:
It seems to me trying to detect shader languages is not necessary and very error prone.

Instead we should depend on the shader profile passed on the 2MGFX command line (in pipeline we set the right profile based on the target platform) and let the errors alert users to a mistake.

As for the future HLSL to GLSL and other translations that will be done via some additional command line option that doesn't exist yet.